README.md

```diff
@@ -837,6 +837,137 @@ OPTIONS:
 --timestamps, -t           [Optional] Shows timestamps on each line in the log output.
 ```
 
+## Using Private Registry Authentication
+
+If you want to use privately hosted container images with ECS, the ECS CLI can store your private registry credentials for you with AWS Secrets Manager and create an IAM role which ECS can use to access the credentials and private images. This allows you to:
+
+* Store private registry credentials within AWS for use with ECS or other services
+* Add the permissions needed to use your registry secrets to a new or existing Task Execution Role
+* Automatically add your private registry credentials to your task definition when running a task or service
+
+Using privately hosted images with the ECS CLI is done in two parts:
+
+1) Create new AWS Secrets Manager secrets and an IAM Task Execution Role with `ecs-cli registry-creds up`
+2) Run `ecs-cli compose up` to create and run a task definition that includes the new resources
```

Comment:
`ecs-cli compose create` is the command that just creates a task definition. But I'd recommend re-wording this to just reference "The compose commands"